pkg/cri/server/image_remove.go

```diff
@@ -33,15 +35,18 @@ import (
 // Remove the whole image no matter the it's image id or reference. This is the
 // semantic defined in CRI now.
 func (c *criService) RemoveImage(ctx context.Context, r *runtime.RemoveImageRequest) (*runtime.RemoveImageResponse, error) {
+	ctx, span := tracing.StartSpan(ctx, makeSpanName("RemoveImage"))
+	defer span.End()
 	image, err := c.localResolve(r.GetImage().GetImage())
 	if err != nil {
+		span.RecordError(err)
 		if errdefs.IsNotFound(err) {
 			// return empty without error when image not found.
 			return &runtime.RemoveImageResponse{}, nil
 		}
 		return nil, fmt.Errorf("can not resolve %q locally: %w", r.GetImage().GetImage(), err)
 	}
-
+	span.SetAttributes(attribute.String("imageID", image.ID))
```

Comment:
Would it be different from `imageRef`?

Reply:
I think so, but I might be wrong

Image object has both ID (digest of image config) and References (repo tag, repo digest)
```
ID: sha256:1123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
References: []string{
			"docker.io/library/busybox:latest",
			"docker.io/library/busybox@sha256:e6693c20186f837fc393390135d8a598a96a833917917789d63766cab6c59582",
		},
```

However, In containerd metadata store, I see that we assign Image.ID to metadata ImageRef https://github.com/containerd/containerd/blob/main/pkg/cri/server/container_create.go#L206. So within containerd, imageRef is treated same as image.ID

Reply:
Thanks. How about using `imageRef` here then? I'd like to keep attributes consistent.

Reply:
okay. I am changing all the attributes to lowercase and dot notation, so this will be "image.ref"